Various boostrap theme fixes

pserwylo opened the pull request:
The commit messages explain the individual fixes, but it should resolve Issue #31, Issue #30, and address (but not fix) Issue #32.

The main change was to move the app details "info box" thing from the right and side to below the app description. Although I liked it there, it caused a whole bunch of troubles. I also think it looks almost equally as nice where it is now.

dervishe- commented:
Works well, below the new state:
![depots petits debrouillards 1](https://cloud.githubusercontent.com/assets/4670759/5325571/90ce4e1e-7cef-11e4-98af-9693367a9a92.png)

pserwylo committed:
Misc fixes to boostrap theme.

 * App details summary info-box now doesn't float right - this fixes
   several annoying issues I was facing.
 * Changed ids from *_version to *_versionCode to not break validator
 * Moved some inline styles to css file.

pserwylo committed:
Wrap text in latest apps list.

Fixes issue #31.

pserwylo committed:
Renamed boostrap.theme.css to theme.css to clear consfusion.

This file is _not_ provided by bootstrap, it is written for this project.
However, the bootstrap.min.css _is_ provided by bootstrap, so will keep
that named as is.
